torch/_meta_registrations.py

```diff
@@ -1189,6 +1189,15 @@ def meta_addbmm(self, batch1, batch2, *, beta=1, alpha=1):
     return self.new_empty(self.size())
 
 
+@register_meta([aten._foreach_add_.List])
```

Comment:
Hey @bdhirsh, is this change look OK to you?

Reply:
Looks ok to me!

One thing - the python meta impl should have the same set of its restrictions as its C++ counterpart. I also see checks that both input tensors are non-empty in the C++ version: https://github.com/pytorch/pytorch/blob/2c16b73a1b310a58a0a1e8544cb60c133e753d6f/aten/src/ATen/native/ForeachUtils.h#L40

Reply:
Good point! Thanks! Will add this.